modules/administration/pages/tshoot-osadjabberd.adoc

```diff
@@ -23,18 +23,25 @@ If more detailed instructions are required, put them in a "Resolving" procedure:
 
 == Open File Count Exceeded
 
-The number of maximum files that a jabber user can open is lower than the number of connected clients.
-OSAD clients cannot contact the SUSE Manager Server, and jabberd requires long periods of time to respond on port 5222.
-Each client requires one permanently open TCP connection and each connection requires one file handler.
+If you receive an ``open file count exceeded`` error, it could be because the maximum number of files that jabber can open is lower than the number of connected OSAD clients.
```

Comment:
I am not even sure this symptom is visible - according to an old bug the only way to see this is via `strace`.

I'd just mention the main visible symptom is that clients cannot be contacted, and the CPU spikes.


https://bugzilla.novell.com/show_bug.cgi?id=950184